src/sentry/spans/consumers/process/factory.py

```diff
@@ -70,8 +70,14 @@ def _build_snuba_span(relay_span: Mapping[str, Any]) -> MutableMapping[str, Any]
     )
 
     trace_context = relay_span.get("data", {}).get("trace")
-    if trace_context:
-        snuba_span["group_raw"] = trace_context.get("hash")
+    snuba_span["group_raw"] = (
+        trace_context.get(
+            "hash",
+            "0",
+        )
+        if trace_context
+        else "0"
+    )
```

Comment:
As discussed: even if Relay forwarded the transaction's trace context (which it doesn't), it would not contain an entry for `"hash"`, because that is written in sentry, not relay: https://github.com/getsentry/sentry/blob/a0b5760f08079efc84751717d7e8ac56d538894b/src/sentry/spans/grouping/result.py#L49-L53

So we might as well shorten this to

```suggestion
    snuba_span["group_raw"] = "0"
```